lib/PAUSE/package.pm

```diff
@@ -707,6 +677,66 @@ Please report the case to the PAUSE admins at modules\@perl.org.},
 
 }
 
+sub __do_regular_perl_update {
+    my ($self, $ctx, $old_row, $arg) = @_;
+
+    my ($opack, $oldversion, $odist, $ofilemtime, $ofile) = @$old_row{
+      qw( package version dist filemtime file )
+    };
+
+    my $older_isa_regular_perl = $arg->{older_isa_regular_perl};
```

Comment:
Okay, I have been putting off serious work on this MR because this was clearly the biggest thing to do: test indexing perls.  I will now attempt a basic set of tests.  Wish me luck.

Reply:
I added new tests (and test helpers) in #510 to test the uncovered code — this and a few other pieces, thanks to [coveralls](https://coveralls.io/jobs/141980705/source_files/25495601820).  Then I rebased this branch on that one and fixed the tests that had been broken.